sdk/lib/rtl/heapuser.c

```diff
@@ -127,3 +127,32 @@ RtlInitializeHeapManager(VOID)
     RtlInitializeCriticalSection(&RtlpProcessHeapsListLock);
 }
 
+ULONG NTAPI
+RtlGetProcessHeaps(_In_ ULONG HeapCount, _Out_cap_(HeapCount) HANDLE *HeapArray)
```

Comment:
```suggestion
RtlGetProcessHeaps(_In_ ULONG HeapCount, _Out_opt_cap_(HeapCount) HANDLE *HeapArray)
```

Reply:
Where is this `_Out_opt_cap_` documented?

Reply:
As with almost all SAL annotations, only in the header (see top results)
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_Out_opt_cap_
It's like `_Out_cap_` but for "optional" pointers (like `_Out_` vs. `_Out_opt_`)